exercises/binary_converter.cpp

```diff
@@ -5,3 +5,25 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#include <iostream> 
+#include <string> 
+ 
+using namespace std; 
+ 
+int main(){ 
+ int n; 
+ string binary_num= ""; 
+  
+ cout<<"Inserisci un numero: "; 
+ cin>>n; 
+ while (n>0) { 
```

Comment:
questa parte andrebbe separata in un metodo a parte che prende "n" come parametro

Reply:
fixed in uno dei commit recenti.